ethexe/runtime/common/src/state.rs

```diff
@@ -111,6 +121,41 @@ pub struct ProgramState {
     pub executable_balance: Value,
 }
 
+impl ProgramState {
+    pub const fn zero() -> Self {
+        Self {
```

Comment:
mb use derive default?

Reply:
I think const fns with more intuitive about usage naming for the case looks better